projects/storefrontlib/src/cms-components/organization/cost-centers/cost-center-form/cost-center-form.component.html

```diff
@@ -1,17 +1,3 @@
-<ng-template
-  #validation
-  let-control="control"
-  let-translationParams="translationParams"
->
-  <span class="is-invalid">
-    <span *ngFor="let error of control.errors | keyvalue">
-      {{
-        'costCenterForm.message.' + error.key | cxTranslate: translationParams
-      }}
-    </span>
-  </span>
-</ng-template>
-
 <div [formGroup]="form">
```

Comment:
This element must be a `<form>` element. Also it should have `(ngSubmit)` event. Please check other forms implementations. When using `(ngSubmit)`, submit button should have `[type=submit]` attribute and no other `(click)` events.

Reply:
Also the name of the form should be related to the component that its used in. Please read https://github.com/SAP/spartacus/blob/develop/docs/migration/2_0.md#forms-changes

Reply:
Also, according to the above docs, please change the submit function. It should check validity of the form and either submit it (do what currently happens) or `this.formName.markAllAsTouched()` - this will trigger FormErrors

Reply:
If you have any required fields, please don't use `required` attribute. Instead, please use `Validators.Required` ng validator.